eng/common/performance/perfhelixpublish.proj

```diff
@@ -152,10 +152,10 @@
       <PayloadDirectory>$(WorkItemDirectory)\ScenarioCorrelation</PayloadDirectory>
       <Command>$(Python) %HELIX_CORRELATION_PAYLOAD%\performance\src\scenarios\crossgen2\test.py crossgen2 --single System.Private.CoreLib.dll --core-root %HELIX_CORRELATION_PAYLOAD%\Core_Root</Command>
     </HelixWorkItem>
-    <HelixWorkItem Include="Crossgen2 Composite Framework R2R">
+    <!-- <HelixWorkItem Include="Crossgen2 Composite Framework R2R">
```

Comment:
It seems like the multiline comment is not working correctly and sendtohelix is failing.

Reply:
I just deleted the worktiem, waiting for the validation build.